*** /etc/rc3.d/S99dtlogin.orig	Wed Aug 13 10:41:24 1997
--- /etc/rc3.d/S99dtlogin	Tue Aug 12 13:54:00 1997
***************
*** 81,87 ****
  	update_printers
  
  	if [ -x /usr/dt/bin/dtlogin ] ; then
! 		/usr/dt/bin/dtlogin &
  	fi
  	;;
  
--- 81,87 ----
  	update_printers
  
  	if [ -x /usr/dt/bin/dtlogin ] ; then
! 		LD_PRELOAD="/usr/lib/libdce.so /usr/lib/libsocket.so /usr/lib/libnsl.so /usr/lib/libthread.so /usr/lib/libm.so /etc/dt/lib/dtlogin-dce.so.1" /usr/dt/bin/dtlogin &
  	fi
  	;;
  
